### DISCUSSION TITLE
how to map "concatenation of outputs from different layers and feed the feature maps to different heads for different outputs in tiny_yolov3_str" from slayer to lava processes

### DISCUSSION THREAD

**Discussion by gwgknudayanga** [lava-nc/lava-dl#337] [Q&A]
Hello,

After training tiny_yolov3_tiny_str in slayer i need to map it .net file so that it can be deployed in loihi. But in the middle how we should map the concatenation of channels from head2_backend and backend ?
In slayer it is as following pytorch operation.
head2 = torch.concat([h2_backend, backend], dim=2)

And also tiny-yolov3 get the output from two heads. However those two heads share the same backend upto some level of the network.
So how to feed the same backend to two different heads in loihi2? How we can do this when dumping slayer trained network to .net for lava?

Thank you.
